spec/abstract/AARCH64/Arch_Structs_A.thy

```diff
@@ -369,13 +369,14 @@ text \<open> Arch-specific part of a TCB: this must have at least a field for us
 record arch_tcb =
   tcb_context :: user_context
   tcb_vcpu    :: "obj_ref option"
+  tcb_cur_fpu :: bool
```

Comment:
We should probably have a comment in the spec what this field is used for and that it is a ghost variable (we only write to it, never read from it).

Reply:
(The comment could be mostly copy/paste from the commit message)